crates/hyperswitch_connectors/src/connectors/bambora.rs

```diff
@@ -119,6 +120,27 @@ impl ConnectorCommon for Bambora {
         )])
     }
 
+    fn get_supported_payment_methods(&self) -> Option<SupportedPaymentMethods> {
+        let mut supported_payment_methods = SupportedPaymentMethods::new();
+        let mut card_payment_method = PaymentMethodTypeMetadata::new();
+        card_payment_method.insert(
+            enums::PaymentMethodType::Credit,
+            PaymentMethodDetails {
+                availability_status: enums::PaymentMethodStage::Live,
+                supports_mandates: false,
+            },
+        );
+        card_payment_method.insert(
+            enums::PaymentMethodType::Debit,
+            PaymentMethodDetails {
+                availability_status: enums::PaymentMethodStage::Live,
+                supports_mandates: false,
+            },
+        );
+        supported_payment_methods.insert(enums::PaymentMethod::Card, card_payment_method);
+        Some(supported_payment_methods)
+    }
```

Comment:
define a toml file for the config and get these values from that config file. That will be maintainable